src/pages/audio-rewards-page/ChallengeRewardsTile.tsx

```diff
@@ -133,15 +148,35 @@ const useRewardIds = () => {
   return filteredRewards
 }
 
+const CHALLENGE_REFRESH_INTERVAL_MS = 5000
+
 const RewardsTile = ({ className }: RewardsTileProps) => {
   const setVisibility = useSetVisibility()
   const dispatch = useDispatch()
   const rewardIds = useRewardIds()
   const userChallengesLoading = useSelector(getUserChallengesLoading)
+  const currentStepCountOverrides = useOptimisticChallengeCompletionStepCounts()
+
+  const [haveChallengesLoaded, setHaveChallengesLoaded] = useState(false)
 
   useEffect(() => {
-    dispatch(fetchUserChallenges())
-  }, [dispatch])
+    if (!userChallengesLoading && !haveChallengesLoaded) {
+      setHaveChallengesLoaded(true)
+    }
+  }, [userChallengesLoading, haveChallengesLoaded])
+
+  useEffect(() => {
+    if (!haveChallengesLoaded) {
+      dispatch(fetchUserChallenges())
+    }
+    const interval = setInterval(() => {
+      dispatch(fetchUserChallenges())
+    }, CHALLENGE_REFRESH_INTERVAL_MS)
+
+    return () => {
+      clearInterval(interval)
+    }
+  }, [dispatch, haveChallengesLoaded])
```

Comment:
Might be nice to extract this functionality into a standalone hook with a comment explaining the refresh functionality here